### PR TITLE
Show window on all workspaces by default

### LINKED ISSUE #43: The window opens only on the desktop where the application was opened
Currently if you click on the tray icon on a mac, the mac automatically switches to the desktop where the application was started. This window behaviour is not well suited for a tray application.
I'd like to change the current behaviour by using the `browserWindow.setVisibleOnAllWorkspaces(true)` option. This option is used to show the application on all workspaces.

### PR DESCRIPTION
**Pull request by marcbachmann**
This fixes #43

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -114,7 +114,13 @@ module.exports = function create (opts) {
         menubar.window.setPosition(x, y)
       }
 
-      if (!opts['always-on-top']) menubar.window.on('blur', hideWindow)
+      if (!opts['always-on-top']) {
+        menubar.window.on('blur', hideWindow)
+      }
+
+      if (opts['show-on-all-workspaces'] !== false) {
```
Comment (jenslind) [inline index.js]: IMO we should use camelcase for all options, it is used everywhere else. @maxogden agree?
All options should also be added to the readme. :)

Comment (marcbachmann) [inline index.js]: Yes, we should use camelCase. What about the `always-on-top` option? Should we also rename that and make a major release? Or do you like to support both notations for that option?

Comment (fabien-d) [inline index.js]: Electron uses `-` for options. https://github.com/atom/electron/blob/master/docs/api/browser-window.md.

Comment (jenslind) [inline index.js]: Yeah, and as long as we are consistent I will be happy. :)

Comment (jenslind) [inline index.js]: @maxogden should we go with camelCase or dashes? After we have decided we should make sure all options using the same format. Then I suggest we merge this and #45 and release `v3`.

**Comment by jenslind** [comment]
Whatever, lets keep the dashes for now. :racehorse:

**Comment by jenslind** [comment]
`2.1.2`

**Comment by rhysd** [comment]
:+1:

**Pull request merged**